packages/migration/src/migrations/v0003_fullbleedToFill.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import type {
+  GifElementV0,
+  ImageElementV0,
+  VideoElementV0,
+  ProductElementV0,
+  ShapeElementV0,
+  TextElementV0,
+  PageV0,
+} from '../types';
+import type { PageV2, StoryV2, UnionElementV2 } from './v0002_dataPixelTo1080';
+
+export interface VideoElementV3 extends Omit<VideoElementV0, 'isFullbleed'> {
+  isFill?: boolean;
+}
+
+export interface GifElementV3 extends Omit<GifElementV0, 'isFullbleed'> {
+  isFill?: boolean;
+}
+
+export interface ImageElementV3 extends Omit<ImageElementV0, 'isFullbleed'> {
+  isFill?: boolean;
+}
+
+export type UnionElementV3 =
```

Comment:
@barklund @swissspidy This PR now uses the initial StoryV0 as the base type to extend for each following migration and has a few first files migrated, PTAL.

Let me know what you think of this system with the UnionElement type here. Currently, it's using the last extended version of the specific element, e.g. the ShapeElement has not been changed meanwhile so it still uses `V0`, the VideoElement type has already been changed so it uses the `VideoElementV3`.

An alternative would be creating a new version for each type in each migration file, even if it doesn't change. E.g. each migration file exports a new type for each element type even if the element type doesn't get extended. It would perhaps be more clear to read and easier for the next version even though it would mean adding a bunch of types just for the purposes of having a type for each version.

Also, if we'd do that, should we do it for the "main" types that are most likely used in most of the migration files (different element types, page, etc.), or should we do it for all types (e.g. resource, font metrics, etc. which likely change less or not change at all).

Thoughts?